commandhandlers.go

```diff
@@ -25,6 +25,16 @@ var pingHandler = func(s *state.State, e *gateway.InteractionCreateEvent, option
 	return makeEphemeralResponse(response)
 }
 
+var permissionRequireAdministrator = func() *discord.Permissions {
+	v := discord.PermissionAdministrator
+	return &v
+}
+
+var permissionRequireBanMembers = func() *discord.Permissions {
+	v := discord.PermissionBanMembers
+	return &v
+}
+
```

Comment:
These seem redundant

Reply:
Intention is to have additional users who are capable of banning members have access to `/ban` since the discord server role hierarchy is commonly defined as `Admins, Moderators, etc...`. So the purpose of these separated permissions was to allow server mods to have access to `/ban` while admins takes care of `/config`. Probably less important if this bot is only to be used in niche use cases.

Reply:
That makes sense.